### DISCUSSION TITLE
TailwindCSSをどうにかしたい

### DISCUSSION THREAD

**Discussion by pss-aileen** [first-contributions-ja/first-contributions-ja.github.io#135] [90: 知識・知見の共有]
クラスが長すぎて泣きそうなので
バージョン1リリース後にコードを綺麗にする期間を作って
整えたい
https://tailwindcss.com/docs/reusing-styles#extracting-classes-with-apply

**Comment by kazzyfrog** [comment]
いい感じに整理したいですね！

上記のtailwindの公式ページに、「`@apply`は読みづらさを回避するために利用するべきではない、、」
的なことが書いてあるので、コンポーネントを細かく分けてみても、いいかもです！
